File: cloud/config-server/src/main/java/pers/acp/admin/config/constant/ConfigApi.java
package pers.acp.admin.config.constant;

/**
 * @author zhang by 10/01/2019
 * @since JDK 11
 */
public interface ConfigApi {

    String basePath = "/config";

    String propertiesConfig = "/properties";

    String propertiesRefresh = propertiesConfig + "/refresh";

}

File: cloud/config-server/src/main/kotlin/pers/acp/admin/config/constant/ConfigApi.kt
package pers.acp.admin.config.constant

/**
 * @author zhang by 10/01/2019
 * @since JDK 11
 */
object ConfigApi {
    const val basePath = "/config"
    const val propertiesConfig = "/properties"
    const val propertiesRefresh = "$propertiesConfig/refresh"
    const val propertiesRefreshApplication = "$propertiesRefresh/application"
    const val propertiesRefreshMatcher = "$propertiesRefresh/matcher"
}